src/main/scala/org/broadinstitute/hail/methods/Aggregators.scala

```diff
@@ -28,29 +31,29 @@ object Aggregators {
         aggregatorA(0) = v
         aggregatorA(1) = va
         (gs, localSamplesBc.value, localAnnotationsBc.value).zipped
-          .foreach {
-            case (g, s, sa) =>
-              aggregatorA(2) = s
-              aggregatorA(3) = sa
-              baseArray.foreach {
-                _.seqOp(g)
-              }
+          .foreach { case (g, s, sa) =>
+            aggregatorA(2) = s
+            aggregatorA(3) = sa
+            (baseArray, aggFunctions).zipped.foreach { case (agg, aggF) =>
+              agg.seqOp(aggF(g))
+            }
           }
 
         baseArray.foreach { agg => aggregatorA(agg.idx) = agg.result }
       }
       Some(f)
-    } else None
+    } else
+      None
```

Comment:
Seems right to me.  IntelliJ agrees.  I much prefer this style.  I think an `if` should be 1 line, or then and else clauses should be on different lines.  I don't really like

```
  if (p) a
  else b
```

and I *really* don't like

```
  if (p) {
    a
  } else b
```